gammapy/stats/tests/test_variability.py

```diff
@@ -164,11 +164,16 @@ def temp(x, norm, index):
         temp, 15, 1 * u.h, power_spectrum_params=params
     )
 
+    time_series4, time_axis4 = TimmerKonig_lightcurve_simulator(
```

Comment:
Looking at the code, there is a `if leakage_protection > 1`. So, should we have 2 different tests?